src/ProjectTemplates/Web.ProjectTemplates/content/BlazorWeb-CSharp/BlazorWeb-CSharp/Components/Identity/StatusMessage.razor

```diff
@@ -0,0 +1,29 @@
+﻿@using BlazorWeb_CSharp.Identity
+
+@{
+    var message = Message ?? MessageFromCookie;
+
+    if (MessageFromCookie is not null)
+    {
+        HttpContext.Response.Cookies.Delete(IdentityRedirectManager.StatusCookieName);
+    }
+}
```

Comment:
Why have this procedural logic during rendering? This is something we tell people to avoid.

Can we move this to `OnInitialized`?